### PR TITLE
Subscribe users to Mailchimp list automatically

### PR DESCRIPTION
**Pull request by sophieH29**

### CODE REVIEW

File: functions/config.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+    "mailchimp": {
```
Comment (ozasadnyy) [inline functions/config.json]: what about using environment  variables https://firebase.google.com/docs/functions/config-env ?

Comment (sophieH29) [reply functions/config.json]: I don't like the idea to fill values manually through cmd, it's not obvious to others who gonna use it.

Comment (ozasadnyy) [reply functions/config.json]: but it's more secure. In this way, people will commit their configs to public repositories. It's much better to write documentation on that

Comment (sophieH29) [reply functions/config.json]: ok, will move there

File: functions/src/mailchimp-subscribe.js [hunk 2]
```diff
@@ -18,12 +22,12 @@ export const mailchimpSubscribe = functions.database.ref('/subscribers/{id}')
...
         };
 
-        subscribeToMailchimp('POST', subscriberData, (bodyObj) => {
+        return subscribeToMailchimp('POST', mailchimpConfig, subscriberData, (bodyObj) => {
             if (bodyObj.status === 400 & bodyObj.title === 'Member Exists') {
                 subscriberData.status = 'pending';
                 const hash = md5(subscriberData.email_address);
 
-                subscribeToMailchimp('PATCH', subscriberData, (bodyObj) => {
+                subscribeToMailchimp('PATCH', mailchimpConfig, subscriberData, (bodyObj) => {
```
Comment (ozasadnyy) [inline functions/src/mailchimp-subscribe.js]: add `return`

Comment (sophieH29) [reply functions/src/mailchimp-subscribe.js]: oh, missed it, ok

**Pull request merged**